mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -437,9 +437,14 @@ def set_roi_visibility_flags(self, roi_name: str, visible: bool) -> None:
         self.show_visible_spectrums()
 
     def show_visible_spectrums(self) -> None:
```

Comment:
As this method does a new things now, can we add a docstring to explain what the method does.